wordpress-mobile/gutenberg-mobile
# Improve Buildkite pipeline by addressing feedback from #3758

This PR addresses the feedback from #3758.
* a0e6f8fd6020b46c54e0e41e0f17e2b92122259c addresses [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3758#discussion_r677461249)
* 0b97c0226caa0ef66918ee28523427069bfc456c addresses [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3758#discussion_r677470806)
* a85a0487a308ef89f9beb1af8c541b132d1582c3 addresses [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3758#discussion_r677168249)

To test:
* Since Buildkite was able to successfully publish the artifacts, we just need to ensure the editor loads without a crash using the [WPAndroid installable build](https://github.com/wordpress-mobile/WordPress-Android/pull/15089#issuecomment-888029063).

PR submission checklist:
